collision.cpp

```diff
@@ -324,8 +324,8 @@ void CollideUnitSphereWithTri(CollisionInfo* ci, Tri tri)
             ci->nearestDistance = distanceToHitpoint;
             ci->hitPoint = hitPoint;
             if (embeddedInPlane) {
-                //ci->nearestDistance = -VERY_CLOSE_DIST;
-                //ci->hitPoint -= VERY_CLOSE_DIST*p.normal;
+                //ci->nearestDistance = -DOD_VERY_CLOSE_DIST;
+                //ci->hitPoint -= DOD_VERY_CLOSE_DIST*p.normal;
```

Comment:
is it on purpose that this is commented out?

Reply:
Let me check this. I don't know.

Reply:
Yes. This is just a reminder that maybe we can do something about getting stuck in a plane that has both sides defined as 'front face'. I will add a comment. But it is likely not to work out this way because an infinite small plane is just numerically very difficult ;)